control/commands_repository.go

```diff
@@ -0,0 +1,10 @@
+package control
+
+// CommandsRepository - Data type to store the commands to be sent
+type CommandsRepository []StandardCommand
```

Comment:
Porque não o map mesmo?

Reply:
O id do robô é variável no comando, se ele for o índice no mapa pode ocorrer de, quando for alterado, perder a referência.

Reply:
Saquei, bem colocado. Só tem que garantir que o índice do _slice_ não tem relação com o ID mesmo. O índice na verdade não vai indicar nada então?

Reply:
Em uma primeira ideia, quando a comunicação serial for implementada, o índice no slice vai corresponder ao rádio que deve ser enviado o pacote, pois vamos usar um rádio para cada robô. Caso isso não for aplicável, o índice do slice realmente não vai indicar nada.

Reply:
Tá, se o indice tiver dependencia aí sim eu vou recomendar um _map_, mas enquanto ele não importar pra nada um _slice_ é de fato a melhor escolha.